### ISSUE TITLE
``Generate Lighthouse Accessibility Score`` CI is failing

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#1655]
https://github.com/NationalSecurityAgency/skills-service/actions/workflows/generate-lighthouse-score.yml
